### PR TITLE
update docs for user-data

### PR DESCRIPTION
**Pull request by mrby**
changing to match https://deepstream.io/tutorials/core/auth-file/

### CODE REVIEW

File: src/docs/server/user-file/readme.md [hunk 1]
```diff
@@ -75,15 +81,14 @@ e.g.
 ![deepstream hash console output](ds-hash-output.png)
 
 ### data
-Optional authentication data, e.g. `role: admin` or `canCreatePosts: true`. This data will be available in your permission rules as `user.data`. Data stored within clientData will also be forwarded to the client as part of the login process.
```
Comment (AlexBHarley) [inline src/docs/server/user-file/readme.md]: Why was this removed? This is correct. See https://deepstream.io/tutorials/core/permission-conf-advanced/

File: src/docs/server/user-file/readme.md [hunk 2]
```diff
@@ -13,13 +13,17 @@ johndoe:
   # Password as hash if auth.options.has is configured or in cleartext
   password: uY2zMQZXcFuWKeX/6eY43w==9wSp046KHAQfbvKcKgvwNA==
   # Optional auth data that will be passed to permissioning as user.data
-  data:
+  clientData:
```
Comment (yasserf) [inline src/docs/server/user-file/readme.md]: role is server data, not client data.

File: src/docs/server/user-file/readme.md (hunk 1 above)
Comment (yasserf) [inline src/docs/server/user-file/readme.md]: Yup, it confusing but serverData is what becomes user.data in permissions.

File: src/docs/server/user-file/readme.md (hunk 2 above)
Comment (mrby) [inline src/docs/server/user-file/readme.md]: fixed

File: src/docs/server/user-file/readme.md [hunk 3]
```diff
@@ -75,15 +81,14 @@ e.g.
 ![deepstream hash console output](ds-hash-output.png)
 
 ### data
-Optional authentication data, e.g. `role: admin` or `canCreatePosts: true`. This data will be available in your permission rules as `user.data`. Data stored within clientData will also be forwarded to the client as part of the login process.
+Optional authentication data, e.g. `role: admin` or `canCreatePosts: true`. This will be available in your permission rules as `user.serverData`. Data stored within clientData will be forwarded to the client as part of the login process.
```
Comment (yasserf) [inline src/docs/server/user-file/readme.md]: This part is actually super confusing, but it shows up in permission rules as user.data =(

Other than that this looks good!

**Comment by yasserf** [comment]
looks good! thanks
